pulpcore/content/handler.py

```diff
@@ -1127,8 +1127,9 @@ async def finalize():
         except DigestValidationError:
             # Using this otherwise unused dbfield is a workaround to allow this fix to be
             # backported. Read "failed_at" instead.
-            remote_artifact.pulp_last_updated = timezone.now() + timedelta(
-                settings.REMOTE_CONTENT_FETCH_FAILURE_COOLDOWN
+            RemoteArtifact.objects.filter(pulp_id=remote_artifact.pulp_id).update(
+                pulp_last_updated=timezone.now()
+                + timedelta(settings.REMOTE_CONTENT_FETCH_FAILURE_COOLDOWN)
             )
             await remote_artifact.asave()
```

Comment:
```suggestion
            await RemoteArtifact.objects.filter(pulp_id=remote_artifact.pulp_id).aupdate(
                pulp_last_updated=timezone.now()
                + timedelta(settings.REMOTE_CONTENT_FETCH_FAILURE_COOLDOWN)
            )
```
You cannot use sync db calls in an async context.

Reply:
Yeah, was just looking into that. Thanks